code/code/misc/discord.h

```diff
@@ -13,5 +14,6 @@ class Discord {
         static int ACHIEVEMENT_THRESHOLD;
 
         static bool doConfig();
-        static bool sendMessage(sstring channel, sstring msg);
+        static bool sendMessageAsync(sstring channel, sstring msg);
```

Comment:
Is there a reason why this thing has to be publicly visible? Implementation details preferably wouldn't be exposed.

Reply:
oversight, good catch!